docs/docs/guide/debugging_tables/tables.tsx

```diff
@@ -0,0 +1,262 @@
+import React from 'react';
+
+/*
+  Testing setup:
+    - Shared:
+      - react-native: 0.69.3
+      - empty app generated using the typescript template
+      - added reanimated only
+      - tested: scrollTo, measure, useAnimatedSensor, console logs with _WORKLET
+    - Android:
+      - Device: Pixel 5 API 32
+    - iOS:
+      - Device: iPhone 13 Pro (iOS 15.5)
+*/
+
+export function SummaryTable() {
+  return (
+    <>
+      <table>
+        <thead>
+          <tr>
+            <th style={{ width: '30%' }}>Tool</th>
+            <th style={{ width: '10%' }}>Platform</th>
+            <th style={{ width: '20%' }}>JSC</th>
+            <th style={{ width: '20%' }}>Hermes</th>
+            <th style={{ width: '20%' }}>V8</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td rowSpan={2}>
+              <a href="#chrome-debugger">Chrome debugger</a>
+            </td>
+            <td style={{ textAlign: 'center' }}>Android</td>
+            <td style={{ textAlign: 'center' }}>✅ ¹</td>
+            <td style={{ textAlign: 'center' }}>✅ ¹</td>
+            <td style={{ textAlign: 'center' }}>✅ ¹</td>
+          </tr>
+          <tr>
+            <td style={{ textAlign: 'center' }}>iOS</td>
+            <td style={{ textAlign: 'center' }}>✅ ¹</td>
+            <td style={{ textAlign: 'center' }}>✅ ¹</td>
+            <td style={{ textAlign: 'center' }}>N/A</td>
```

Comment:
Can we add `color: lightgray;` for `N/A` cells? It looks like this:

<img width="521" alt="lightgray" src="https://user-images.githubusercontent.com/20516055/185056271-a3121e1a-0786-4c7a-be1b-2e56af36c2a5.png">